content/docs/command-reference/remote/add.md

```diff
@@ -162,36 +162,27 @@ For more information about the variables DVC supports, please visit
 
 ```dvc
 $ dvc remote add -d myremote azure://mycontainer/path
-$ dvc remote modify --local myremote connection_string 'mystring'
+$ dvc remote modify --local myremote connection_string 'mysecret'
 ```
 
-> The connection string contains sensitive user info. Therefore, it's safer to
-> add it with the `--local` option, so it's written to a Git-ignored config
-> file. See `dvc remote modify` for a full list of Azure parameters.
+> The Azure Storage
+> [connection string](http://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/)
+> contains sensitive user info. Therefore, it's safer to add it with the
+> `--local` option, so it's written to a Git-ignored config file. See
+> `dvc remote modify` for a full list of Azure parameters.
 
 The Azure Blob Storage remote can also be configured globally via environment
 variables:
 
 ```dvc
 $ export AZURE_STORAGE_CONNECTION_STRING='mysecret'
-$ export AZURE_STORAGE_CONTAINER_NAME='mycontainer'
-$ dvc remote add -d myremote 'azure://'
+$ dvc remote add -d myremote azure://mycontainer/path
```

Comment:
I'm not sure whether we'd want to document this or keep as an internal behavior but just to note: unlike s3/google storage buckets, when the containers doesn't exist we automatically create them.

Reply:
I added a note in `remote modify`.